WoWsShipBuilder.ViewModels/Helper/GridData/BombersGridDataWrapper.cs

```diff
@@ -0,0 +1,129 @@
+﻿using WoWsShipBuilder.Core.DataContainers;
+
+namespace WoWsShipBuilder.ViewModels.Helper.GridData;
+
+public class BomberGridDataWrapper
+{
```

Comment:
A lot of properties in this class are identical in TorpedoBomber and RocketPlane data wrappers. We should consider extracting them into a common base class.
See details on Sonar: https://sonarcloud.io/component_measures?id=WoWs-Builder-Team_WoWs-ShipBuilder&pullRequest=215&metric=new_duplicated_lines_density&view=list